lisa/tools/taskset.py

```diff
@@ -10,9 +12,22 @@ class TaskSet(Tool):
     def command(self) -> str:
         return "taskset"
 
+    @classmethod
+    def _freebsd_tool(cls) -> Optional[Type[Tool]]:
+        return BSDTaskSet
+
     @property
     def can_install(self) -> bool:
         return False
 
     def run_on_specific_cpu(self, cpu_id: int) -> Process:
         return self.run_async(f"-c {cpu_id} yes > /dev/null", shell=True)
+
+
+class BSDTaskSet(TaskSet):
+    @property
+    def command(self) -> str:
+        return "cpuset"
+
+    def run_on_specific_cpu(self, cpu_id: int) -> Process:
+        return self.run_async(f"-l {cpu_id} yes > /dev/null", shell=True)
```

Comment:
if this method is the same, we can reuse above one?

Reply:
The method I wrote uses a different flag from the above one. I could rewrite it so that the above method checks the OS and dynamically changes the flag but I think this is a cleaner and more easily readable format.